service/pokemon_service.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"errors"
+
+	"github.com/ErickRodriguezWize/academy-go-q42021/domain/model"
+)
+
+var (
+	PokemonMissingError = errors.New("Couldn't find pokemon")
+)
```

Comment:
Use a var block if declaring more than one variable, otherwise, just use var.

I'd recommend not exporting this if you are not going to use it in another place.
```suggestion
var pokemonMissingError = errors.New("Couldn't find pokemon")
```